modules/components/pages/inputs/sql_select.adoc

```diff
@@ -108,29 +108,27 @@ input:
 
 A database <<drivers, driver>> to use.
 
-
 *Type*: `string`
 
-
 Options:
-`mysql`
-, `postgres`
-, `clickhouse`
+`clickhouse`
```

Comment:
Is this omitted comma intentional?

Reply:
Hi @mihaitodor, I've looked at the preview and can't see a missing comma: [https://deploy-preview-122--redpanda-connect.netlify.app/redpanda-connect/components/inputs/sql_select/#driver](https://deploy-preview-122--redpanda-connect.netlify.app/redpanda-connect/components/inputs/sql_select/#driver). Do you agree?

Reply:
Oh, my bad! I didn't realise `clickhouse` ended up being the first item in the list. Thank you for checking! ❤️